cax/tasks/data_mover.py

```diff
@@ -592,10 +592,20 @@ def copy_tsm(self, datum, destination, method, option_type):
           for name in dirs:
             list_folders.append(name)
         
+        #Sanity check from zero sized files during the upload
+        # find *.pickles or *log files which are size of zere and delte them before upload
```

Comment:
This comment says you check if file is ```*.pickles``` or ```*log```, but I don't see this check below.  For example, a zero-sized ```.zip``` could indicate a problem, but would just get deleted here.

Reply:
This is true. I put this in the comment section to make the purpose clear (*pickle and eventually an empty log file).
For the deletion process later, I would say that a zero sized zip file indicates a problem based on pax. But we also should not upload these files then. For rucio it does not make a difference and it will complain about any zero sized file then.

BTW: I have never seen a zero sized *zip file ever.